auth/src/api/rolesChange.js

```diff
@@ -72,6 +67,8 @@ router.post(
 
     // If the current user has the correct permission level, promote the user or send a corresponding success or error message
     const roles = await getRolesForUser(user.role);
+    roles.push(user.role);
```

Comment:
What's this for?

Reply:
This is to allow admins to elevate the role of other users to their same role, which is admin.